server/src/main/java/org/apache/cassandra/sidecar/acl/authorization/DomainAwarePermission.java

```diff
@@ -45,7 +46,18 @@ public class DomainAwarePermission extends StandardPermission
 
     public DomainAwarePermission(String name)
     {
-        super(name);
+        this(name, SCOPE_LESS);
```

Comment:
I think scopeless is a single word. `-less` is the suffix.

Reply:
changing to `NO_SCOPE` instead